consensus/istanbul/utils.go

```diff
@@ -65,5 +65,12 @@ func CheckValidatorSignature(valSet ValidatorSet, data []byte, sig []byte) (comm
 		return val.Address(), nil
 	}
 
+	// 3. Log signer that is missing in validator set
+	addresses := make([]string, len(valSet.List()))
+	for i, validator := range valSet.List() {
+		addresses[i] = validator.Address().String()
+	}	
+	log.Warn("Signer is missing from validator set", "signer", signer, "validatorSet", addresses)
+
```

Comment:
```suggestion
log.Debug("Signer is missing from validator set", "signer", signer, "validatorSet", fmt.Sprintf("%v", valSet.List())
```

Reply:
i think this can be `Warn` since it is not supposed to reach this state, before the error is returned

Reply:
replacing it to `debug` as discusssed